### PR TITLE
add radio button to photo page

### PR DESCRIPTION
**Pull request by LBHABEBIC**
# What?

# Why?

# Notes

# Next steps

### CODE REVIEW

**Review by erbridge — APPROVED** [review]
Looks good. Needs a rebase to fix the tests.

File: steps/id-and-residency/tenant-photo.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import yesNoNotPresentRadio from "helpers/yesNoNotPresentRadio";
```
Comment (erbridge) [inline steps/id-and-residency/tenant-photo.tsx:1]:
```suggestion
import yesNoNotPresentRadio from "../../helpers/yesNoNotPresentRadio";
```

Comment (LBHABEBIC) [reply steps/id-and-residency/tenant-photo.tsx]: This still seems to br failing circle ci even after rebase.

Comment (erbridge) [reply steps/id-and-residency/tenant-photo.tsx]: Suggest you review your code and see where the issue is.

Comment (erbridge) [reply steps/id-and-residency/tenant-photo.tsx]: Also, your rebase seems to have reverted the changes to add the not present button to the page.

**Pull request closed without merging**